File: matiasmasso/Mat.NET/DT2/DTOGuidNomAmt.vb
Public Class DTOGuidNomAmt
    Inherits DTOGuidNom
    Property Amt As DTOAmt

    Public Sub New(oGuid As Guid)
        MyBase.New(oGuid)
    End Sub

    Public Sub New()
        MyBase.New
    End Sub
End Class

File: matiasmasso/Mat.NET/DTO/DTOGuidNomAmt.cs
using System;

namespace DTO
{
    public class DTOGuidNomAmt : DTOGuidNom
    {
        public DTOAmt Amt { get; set; }

        public DTOGuidNomAmt(Guid oGuid) : base(oGuid)
        {
        }

        public DTOGuidNomAmt() : base()
        {
        }
    }
}
